### PR TITLE
Fix connections loop

### PR DESCRIPTION
**Pull request by eslamoo**

### CODE REVIEW

File: jsplumb-persistence-plugin.js [hunk 1]
```diff
@@ -1,207 +1,208 @@
...
-                        }
-                    }
-                    obj["cssClass"] = overlay.canvas.className;
-                    temp.push(obj);
-                    return temp;
-                })
-            });
-        }
+          connections.push({
+              // path: connector.getPath(), //No need cuz it causing error on saving
```
Comment (nitinsurana) [inline jsplumb-persistence-plugin.js:144]: path needs to be saved for the connections to load back with endpoints

File: jsplumb-persistence-plugin.js [hunk 2]
```diff
@@ -1,207 +1,208 @@
...
+              id: $elem.attr('id'),
+              left: parseInt($elem.css("left"), 10),
+              top: parseInt($elem.css("top"), 10),
+              width: parseInt($elem.css("width"), 10),
+              heigth: parseInt($elem.css("heigth"), 10),
+              html: $elem.html()
+          });
+      });
+      var connections = [];
+      $.each(instance.getAllConnections(), function (idx, connection) {
```
Comment (nitinsurana) [inline jsplumb-persistence-plugin.js:103]: `instance` appears to be global variable which will not be present
We should be saving using the `jsPlumbInstance` passed to the save call.

**Comment by nitinsurana** [comment]
The root cause for missing connections in not the plugin source code.

File: jsplumb-persistence-plugin.js (hunk 1 above)
Comment (eslamoo) [reply jsplumb-persistence-plugin.js:144]: Screenshot-20200630102159-406x96
Gives me this error

**Pull request closed without merging**